src/app/card/info-status-card/info-status-card-config.ts

```diff
@@ -0,0 +1,42 @@
+export class InfoStatusCardConfig {
+  /**
+   * Show/hide the top border, true shows top border, false (default) hides top border
+   */
+  showTopBorder?: boolean;
+
+  /**
+   * Flag to allow HTML content within the info options
+   */
+  htmlContent: boolean;
+
+  /**
+   * Status configuration information
+   */
+  status: {
+
+    /**
+     * The main title of the info status card
+     */
+    title?: string;
+
+    /**
+     * The href to navigate to if one clicks on the title or count
+     */
+    href?: string;
+
+    /**
+     * An icon to display to the left of the count
+     */
+    iconClass?: string;
```

Comment:
Please rename this as inconStyleClass to be consistent with other component properties.